DateTime.js

```diff
@@ -41,6 +42,7 @@ var Datetime = createClass({
 			onFocus: nof,
 			onBlur: nof,
 			onChange: nof,
+                        datetimeFormat: false,
```

Comment:
I'm not sure I like this naming, to me the format we are talking about is for the _value_ in the input field. Datetime is such a broad word. Maybe `valueFormat`, what do you think?